eng/pipelines/aggregate-reports.yml

```diff
@@ -53,8 +34,32 @@ jobs:
       pwsh: true
       workingDirectory: $(Build.SourcesDirectory)
       filePath: eng/common/scripts/Verify-Resource-Ref.ps1
+      
+  - template: /eng/pipelines/templates/steps/analyze_dependency.yml
```

Comment:
What was this order changed? I think this needs to be moved back.

Reply:
Do you mean analyze_dependency should go after run cred scan step?